src/csvcubed/models/cube/qb/validationerrors.py

```diff
@@ -308,3 +308,18 @@ class BothMeasureTypesDefinedError(IncompatibleComponentsError):
     @classmethod
     def get_error_url(cls) -> str:
         return 'http://purl.org/csv-cubed/err/both-meas-typ-def'
+
+
+@dataclass
+class EmptyQbMultiMeasureDimensionError(SpecificValidationError):
+    """
+    An error for when the user has a `QbMultiMeasureDimension` but its measure
+    field is an empty list.
+    """
+
+    @classmethod
+    def get_error_url(cls) -> str:
+        return 'http://purl.org/csv-cubed/err/empty-multi-meas-dimension'
```

Comment:
The documentation pages corresponding to these URIs need to be part of this PR too.

This is the location where they should be created:

https://github.com/GSS-Cogs/csvcubed/tree/main/external-docs/docs/guides/errors/build-command-errors

Once they've been set up, we can create the PURL url mappings.